hbase-client/src/main/java/org/apache/hadoop/hbase/util/JvmPauseMonitor.java

```diff
@@ -62,28 +66,47 @@ public class JvmPauseMonitor {
   public static final String INFO_THRESHOLD_KEY = "jvm.pause.info-threshold.ms";
   private static final long INFO_THRESHOLD_DEFAULT = 1000;
 
+  public static final String PAUSE_MONITOR_ENABLE_KEY = "hbase.client.pause.monitor.enable";
+
+  public static final boolean PAUSE_MONITOR_ENABLE_DEFAULT = false;
+
   private Thread monitorThread;
   private volatile boolean shouldRun = true;
-  private JvmPauseMonitorSource metricsSource;
+  private final JvmPauseMonitorSource metricsSource;
+
+  public static synchronized JvmPauseMonitor getInstance(Configuration conf) {
```

Comment:
Why change it to singleton? What if we pass different Configuration here?

Reply:
There are two reasons.

@wchevreuil mentioned that our user may open multiple connections, however only one pause monitor is enough for a process. More details please see [here](https://github.com/apache/hbase/pull/5319#discussion_r1258120860)

Reply:
If we only want one JvmPauseMonitor in process, then we should not bind it to a connection? Just add a utility method to enable jvm pause monitor is enough?

Reply:
Ok. Let me try to fix this. Thanks Duo ! @Apache9